CI Pipeline Fails to Publish Appx

https://dev.azure.com/ms/react-native-windows/_build/results?buildId=173035&view=logs&j=0b7bc0b1-3390-5293-5a9c-af2a4f48175a&t=aa9d9956-6ab4-51e6-4386-c0420d719046&l=10
```
Starting: Upload App Package
==============================================================================
Task         : Publish build artifacts
Description  : Publish build artifacts to Azure Pipelines or a Windows file share
Version      : 1.186.0
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/publish-build-artifacts
==============================================================================
##[error]Publishing build artifacts failed with an error: Not found PathtoPublish: D:\a\_work\1\s\packages\playground\windows\AppPackages\playground
Finishing: Upload App Package
```

@chiaramooney made recent changes to AppX signing logic in this project, which may be related.

--- a/.ado/jobs/playground.yml
+++ b/.ado/jobs/playground.yml
@@ -18,11 +18,13 @@ jobs:
           BuildPlatform: x86
           SolutionFile: Playground.sln
           TestWACK: true
+          UploadAppx: true
         X64ReleaseUniversal:
           BuildConfiguration: Release
           BuildPlatform: x64
           SolutionFile: Playground.sln
           TestWACK: true
+          UploadAppx: true
         X86DebugWin32:
           BuildConfiguration: Debug
           BuildPlatform: x86
@@ -36,6 +38,7 @@ jobs:
           BuildPlatform: x86
           SolutionFile: Playground.sln
           BuildWinUI3: true
+          UploadAppx: true
     pool: $(AgentPool.Medium)
     timeoutInMinutes: 60
     cancelTimeoutInMinutes: 5
@@ -145,7 +148,7 @@ jobs:
 
       - task: PublishBuildArtifacts@1
         displayName: Upload App Package
-        condition: and(succeededOrFailed(), eq('${{ parameters.BuildEnvironment }}', 'Continuous'))
+        condition: and(succeededOrFailed(), eq(variables.UploadAppx, true), eq('${{ parameters.BuildEnvironment }}', 'Continuous'))
         inputs:
           pathtoPublish: 'packages/playground/windows/AppPackages/playground'
           artifactName: 'App Packages $(BuildPlatform) $(BuildConfiguration) ($(System.JobAttempt))'

